tests/Doctrine/Tests/DBAL/Types/VarDateTimeTest.php

```diff
@@ -20,7 +20,10 @@ class VarDateTimeTest extends DbalTestCase
     protected function setUp(): void
     {
         $this->platform = $this->createMock(AbstractPlatform::class);
-        $this->type     = new VarDateTimeType();
+        $this->platform->method('getDateTimeTzFormatString')->willReturn('Y-m-d H:i:s');
+        $this->platform->method('getDateTimeFormatString')->willReturn('Y-m-d H:i:s');
```

Comment:
Please see if the changes from https://github.com/doctrine/dbal/pull/4682 could be backported instead.